content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -0,0 +1,383 @@
+## GLTF Animation Importing
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+Bevy's GLTF importer was extended to import GLTF animations into the new [`AnimationPlayer`] system. This supports both "skeletal animation" and arbitrary transform animations.
+
+<video controls loop><source  src="fox.mp4" type="video/mp4"/></video>
+
+```rust
+struct FoxAnimations {
+    walk: Handle<AnimationClip>,
+}
+
+fn setup(mut commands: Commands) {
+    commands.spawn_scene(asset_server.load("models/animated/Fox.glb#Scene0"));
+    commands.insert_resource(FoxAnimations {
+        walk: asset_server.load("models/animated/Fox.glb#Animation0"),
+    });
+}
+
+fn play_on_load(
+    animations: Res<FoxAnimations>,
+    mut players: Query<&mut AnimationPlayer, Added<AnimationPlayer>>,
+) {
+    for mut player in players.iter_mut() {
+        player.play(animations.walk.clone()).repeat();
+    }
+}
+```
+
+## Unlimited* Point Lights
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @robtfm</div>
+
+Bevy can now render scenes with arbitrary numbers of point lights on platforms that support storage buffers (which is basically everything but WebGL). In the last Bevy release (0.6) we added [Clustered Forward Rendering](/news/bevy-0-6/#clustered-forward-rendering), which is a rendering technique that optimizes each fragment's light calculation costs by assigning lights to "clusters". However in the interest of platform compatibility (WebGL), we initially limited ourselves to 256 lights because that is what fit in a uniform buffer binding.
+
+In **Bevy 0.7**, we added the ability to automatically "upgrade" to using unbounded storage buffers for Clustered Forward Rendering on platforms that support them, enabling unlimited* point lights. There is an asterisk there because in practice this is limited by memory and hardware constraints. 
+
+## Light Clustering Features and Optimizations
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @robtfm, @dataphract, @cart</div>
+
+With the upper limit of 256 point lights removed, the only limit on lights is what the hardware can support and bottlenecks in our algorithms. To increase the number of lights, we made a number of optimizations to our clustering algorithms.
+
+* **Dynamic Light Clusters**: By default cluster x/y slices are now dynamically configured based on the lights in the scene, which can significantly increase performance in some scenes. The clustering behavior is now also user-configurable as FixedZ (the new default dynamic x/y behavior), custom fixed x/y/z slice values, single-cluster, and "no clustering".
+* **Light Frustum Change Detection**: We now use Bevy ECS's change detection feature to only recalculate the view frustum of lights that have changed.
+* **Cluster Assignment Optimizations**: The cluster assignment data access patterns and data structures received a variety of tweaks that improved performance.
+
+// TODO: Show many lights video here
```

Comment:
Markdown doesn't have a comment syntax. The closest thing you can get is "abusing link anchor syntax" as `[]:#(comment)`, if you want to have some markdown that will not output anything to the rendered files. Using HTML comments as `<!-- comment -->` means they still get passed through / outputted to the resulting HTML files, though the browser would, ofc, treat them as comments and ignore them.

In this case, none of that should matter, as these are TODO notes on a WIP document. If anything, using `//` is nice, because it is *not* treated as a comment, meaning that people reviewing the WIP PR using a rendered HTML view can see them and read them, which is useful.

Reply:
Yeah I did "not actual comments" on purpose to make them appear in the rendered document.